PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-mongodb.yml

```diff
@@ -0,0 +1,217 @@
+## Override the namespace for resource deployed by the chart, but can itself be overridden by the local namespaceOverride
+#   namespaceOverride: my-global-namespace
+
+image:
+  ## Bitnami MongoDB registry
+  ##
+  registry: docker.io
+  ## Bitnami MongoDB image name
+  ##
+  repository: bitnami/mongodb
+  ## Bitnami MongoDB image tag
+  ## ref: https://hub.docker.com/r/bitnami/mongodb/tags/
+  ##
+  tag: 4.4.2-debian-10-r0
+  ## Specify a imagePullPolicy
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+
+## MongoDB architecture. Allowed values: standalone or replicaset
+##
+architecture: replicaset
+
+## Use StatefulSet instead of Deployment when deploying standalone
+##
+useStatefulSet: false
+
+## MongoDB Authentication parameters
+##
+auth:
+  ## Enable authentication
+  ## ref: https://docs.mongodb.com/manual/tutorial/enable-authentication/
+  ##
+  enabled: true
+  ## MongoDB root password
+  ## ref: https://github.com/bitnami/bitnami-docker-mongodb/blob/master/README.md#setting-the-root-password-on-first-run
+  ##
+  rootPassword: "<root-password>"
+
+## initdb scripts
+## Specify dictionary of scripts to be run at first boot
+##
+## !!! Only required when using API Umbrella !!!
+## This creates the required database and user for API Umbrella.
+## Make sure to set the DB user configuration and database
+## for the API Umbrella deployment.
+##
+## If you omit the creation of database and user during deployment, make sure
+## to create this database and user afterwards manually before proceeding with the
+## API Umbrella deployment.
+##
+initdbScripts:
+  init_db.js: |
+    // Setup DB for API Umbrella
+    db = db.getSiblingDB('api_umbrella');
+    db.createUser(
+    {
+      user: "api_umbrella",
+      pwd: "api-umbrella-password",
+      roles: [
+        {
+          role: "readWrite",
+          db: "api_umbrella"
+        }
+      ]
+    });
+tls:
+  ## Enable or disable MongoDB TLS Support
+  enabled: false
+  ##
+  ## Bitnami Nginx image
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/nginx
+    tag: 1.19.4-debian-10-r19
+    pullPolicy: IfNotPresent
+
+## Name of the replica set
+## Ignored when mongodb.architecture=standalone
+##
+replicaSetName: rs0
+
+## Enable DNS hostnames in the replica set config
+## Ignored when mongodb.architecture=standalone
+## Ignored when externalAccess.enabled=true
+##
+replicaSetHostnames: true
+
+## MongoDB containers' resource requests and limits.
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
```

Comment:
Lets take out the resources. For a small test instance, this is quite some reosuces:)

Reply:
ok